### PR TITLE
edits to README for clarity

### PR DESCRIPTION
**Pull request by edwardsb3**
Just minor sentence structure improvements, removal of unnecessary punctuation, and addition of a link for Valgrind to match other technology references.

### CODE REVIEW

**Comment by alexhultman** [comment]
I think you saved everything with Windows line endings

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ wss.on('connection', function (ws) {
 });
 ```
 ### C++
-For maximum performance and memory scaling the native interface is recommended. Look in the examples folder for threading and load balancing examples. There is no documentation written yet but a bright person like you will have no problem just reading the header file.
+The native interface is reccommended for maximum performance and memory scaling. Reference the examples folder for threading and load balancing examples. Documentation has not been written yet, but it is soon to come. For now, please reference the header file.
```
Comment (alexhultman) [inline README.md]: typo

**Comment by SEAPUNK** [comment]
What's with the whole "+10,235" part of the PR? Wonky merge or something?

**Comment by edwardsb3** [comment]
My first commit had Windows line endings. I corrected the line endings in the second commit.

**Comment by SEAPUNK** [comment]
Oh, I see.

**Comment by alexhultman** [comment]
You need to unstage everything except the readme

**Comment by silverwind** [comment]
I think a simple squash might do it.

http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by alexhultman** [comment]
I'm closing this, I won't merge something with 10k lines changed.

**Pull request closed without merging**